spec/rails_helper.rb

```diff
@@ -27,6 +27,11 @@
 # If you are not using ActiveRecord, you can remove this line.
 ActiveRecord::Migration.maintain_test_schema!
 
+def mock_sign_in(user)
```

Comment:
I don't think this is wrong but usually we use helper modules for this.
Check [this](https://gist.github.com/HeroicEric/9935542) for a how-to